### PR TITLE
Readme Tweaks

### PR DESCRIPTION
**Pull request by prafulrana**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,13 +27,13 @@ Use this command to change the active Github organization in your session.
 
 ### runnable ssh [name]
 
-Use this command to spawn a shell session into the container. You can specify the name of the container you are trying to reach in the following format: <reponame>/<branchname>.
+Use this command to spawn a shell session into the container. You can specify the name of the container you are trying to reach in the following format: < reponame >/< branchname >.
```
Comment (bkendall) [inline README.md]: may I suggest changing this to:

```
`<reponame>/<branchname>`
```

File: README.md [hunk 2]
```diff
@@ -45,6 +45,6 @@ Use this command to get an overview of all the different containers you have und
 
 You can drill down and list all the branches of a repository by specifying a repository name.
 
-### runnable upload <file>
+### runnable upload < file >
```
Comment (bkendall) [inline README.md]: I don't think you really need the spaces here... it's inconsistent anyway with the rest of the doc

Comment (prafulrana) [inline README.md]: The words disappear if I remove the spaces.

File: README.md (hunk 1 above)
Comment (prafulrana) [inline README.md]: Try it out, the text disappears :/

**Comment by bkendall** [comment]
@prafulrana I figure I just edit it and find out. I assume it's github trying to escape 'dangerous html tags' or something. `<tags>` fixes that :)

**Comment by bkendall** [comment]
merged, albeit manually. :)

**Pull request closed without merging**